deployment/terraform/modules/ec2/main.tf

```diff
@@ -347,3 +347,21 @@ resource "aws_elb" "https" {
     unhealthy_threshold = 4
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "cpu_utilization" {
+  alarm_name                = "cpu-utilization-limit-exceeded"
```

Comment:
Alarm name must be unique within AWS account, and also should be descriptive.
`alarm_name = "${var.resource_name_prefix}-app-server-cpu-utilization"`